testing/gcp_util.py

```diff
@@ -128,14 +128,20 @@ def basic_auth_is_ready(url, username, password, wait_min=15):
   end_time = datetime.datetime.now() + datetime.timedelta(
       minutes=wait_min)
   while datetime.datetime.now() < end_time:
-    sleep(2)
+    sleep(10)
```

Comment:
Why is the sleep at the start of loop? Doesn't this introduce an unnecessary delate?

Reply:
this was copied from iap_is_ready.  and I think it's fine because the first we try to connect to the endpoint is almost unlikely to be ready, anyway.

Reply:
Please move the sleep to the end of the loop. I think that makes more sense.

Reply:
updated.